src/test/resources/jsonld/BT000041593.json

```diff
@@ -3,47 +3,36 @@
   "contribution" : [ {
     "type" : [ "Contribution" ],
     "agent" : {
-      "id" : "https://d-nb.info/gnd/10880724X",
```

Comment:
We seem to be losing some information here in `agent`, is this expected?

Reply:
As @TobiasNx noted this change comes from the update of the test data.